packages/types/src/sdk/koa.ts

```diff
@@ -7,7 +7,15 @@ export interface ContextUser extends User {
   license: License
 }
 
-export interface BBContext extends Context {
+export interface BBContext {
   user?: ContextUser
-  body: any
+  status?: number
+  request: {
+    body: any
+  }
+  params: any
+  body?: any
+  redirect?: any
+  attachment: any
+  throw: any
```

Comment:
We shouldn't need any of these, we `extend Context` so that we get the types provided directly from `koa`.

Reply:
I had to remove the extending of Context, I was getting errors all over the place actually using the `BBContext` - for some reason it was baffling the TS transpiler.

Reply:
In this case I think you should switch out the `BBContext` for an `any` for now. We should jump on a call to try to resolve why it wasn't compiling for you, as I'm able to get full access to the correct koa types locally